core/src/ecs/system/dispatcher.cpp

```diff
@@ -220,61 +250,126 @@ bool Dispatcher::dfsVisit(DFSNode& node, std::vector<DFSNode>& nodes)
         node.m = DFSNode::BLACK;
         if (node.s != nullptr)
         {
-            mSystems.push_back(node.s);
+            // mSystems.push_back(node.s);
+            // CUBOS_WARN("{}", node.s->grouptag);
+            mMainStep->queueSystem(node.s->grouptag, node.s);
         }
         return false;
     }
     }
     return false;
 }
 
-void Dispatcher::callSystems(CommandBuffer& cmds)
+void Dispatcher::SystemStep::call(CommandBuffer& cmds, std::vector<ecs::System<bool>>& mConditions,
+                                  std::bitset<CUBOS_CORE_DISPATCHER_MAX_CONDITIONS> mRunConditions,
+                                  std::bitset<CUBOS_CORE_DISPATCHER_MAX_CONDITIONS> mRetConditions)
```

Comment:
Remove the `m` from the arguments. The `m` prefix is used to indicate that its a field, but here the variables are arguments.